velox/exec/Driver.h

```diff
@@ -277,6 +277,7 @@ struct DriverCtx {
   std::shared_ptr<Task> task;
   Driver* driver{nullptr};
   facebook::velox::process::ThreadDebugInfo threadDebugInfo;
+  std::unordered_set<int32_t> tracedOperatorIds;
```

Comment:
```
/// Tracks the traced operator ids. It is also used to avoid tracing the auxiliary operator such as the aggregation operator used by the table writer to generate the columns stats.
```